### ISSUE TITLE
Add Decimal Points For The FPS/CPS Label

### ISSUE THREAD

**Issue by BrownieGamingX5** [absoIute/Mega-Hack-Pro-Future#470]
Pretty self-explanatory.

**Comment by Brittank88** [comment]
Maybe you can also choose how many decimal places to go to (I.e., 0-whatever)?

**Comment by NewAlexGamer** [comment]
Decimal for fps or cps, does that have any use? I can see why cps, but frames?

**Comment by Nicko2102** [comment]
i'n not sure why fps would be useful, but if it's possible, i see no reason to implement it alongside decimal for cps. 

> Maybe you can also choose how many decimal places to go to (I.e., 0-whatever)?

you were probably thinking of this, but i would suggest to put it in the options list the same way as it's done for accurate percentage

**Comment by MainConMan** [comment]
CPS decimals might make it harder to fake a CPS counter in completions.

**Comment by absoIute** [comment]
FPS is sure possible but I'm not sure how that'd work for the CPS counter, since it's literally just the amount of clicks in the last 1000ms, though what would be the use?

**Comment by MainConMan** [comment]
Maybe it could be the amount of clicks in the past 10 seconds divided by 10. So for example, if you click 35 times in 10 seconds, your CPS would say "3.5". Is there any reason why this wouldn't work well?

**Comment by MainConMan** [comment]
I might be able to think of a better solution, I have a few ideas, but I'll think about that when I get home! :)

**Comment by Nicko2102** [comment]
> Maybe it could be the amount of clicks in the past 10 seconds divided by 10. So for example, if you click 35 times in 10 seconds, your CPS would say "3.5". Is there any reason why this wouldn't work well?

It would be very inaccurate. for example, if you don't click at all for like 20 seconds straight, then click 20 times in the next second, then it will display your CPS as 2, when in reality it should be 20. then if you stop clicking after that one second of clicking, it will display 2 CPS for the next 10 seconds while you're doing nothing.

**Comment by absoIute** [comment]
Yeah, it's something I tried originally when making the counter but averaging it just doesn't give much useful information, will probably just implement this for the FPS counter

**Comment by Nicko2102** [comment]
> You literally can't click 1.5 times. 

This is true, but he's talking about the CPS, not the total clicks. It is entirely possible to click 1.5 times per second.

**Comment by BrownieGamingX5** [comment]
Fine, just do the FPS counter. I want this to be accepted now. CPS counter could be in the future.
